### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by scarlett2018**
The diagram is currently partial clickable, need @yangou1988 's help to tune the diagrams a little bit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,7 +90,19 @@ OpenPAI provides end-to-end manuals for both cluster users and administrators.
...
+| ----------------- | ----------------------- | ---------------------- |
+| Architecture      | Kubernetes + Hadoop YARN| Kubernetes             |
+| Scheduler         | YARN Scheduler          | HiveD / K8S default    |
+| Job Orchestrating | YARN Framework Launcher | Framework Controller   |
+| RESTful API       | v1 + v2                 | pure v2                |
+| Storage           | Team-wise storage plugin| PV/PVC storage sharing |
+| Marketplace       | Marketplace v2          | openpaimarketplace     |
+| SDK               | Python                  | JavaScript / TypeScript|
 
     *If there is any question during deployment, please check [installation FAQs and troubleshooting](https://openpai.readthedocs.io/en/latest/manual/cluster-admin/installation-faqs-and-troubleshooting.html) first. If it is not covered yet, refer to [here](#get-involved) to ask question or submit an issue.*
```
Comment (abuccts) [inline README.md:107]: wrong indent

File: README.md [hunk 2]
```diff
@@ -9,21 +9,35 @@
...
+**OpenPAI [v1.0.0](./RELEASE_NOTE.md#2020-v1.0.0) has been released!**
+
+With the release of v1.0, OpenPAI is switching to a more robust, more powerful and lightweight architecture. OpenPAI is also becoming more and more modular so that the platform can be easily customized and expanded to suit new needs. OpenPAI also provides many AI user-friendly features, making it easier for end users and administrators to complete daily AI tasks. 
+
+<br/>
+ <table>
+        <tr>
+         <td colspan="3" align="center"><strong>Click to Explore the Sub Components/Repositories of OpenPAI</strong></td>
+         </tr>
+        <tr>
```
Comment (abuccts) [inline README.md:21]: indent

File: README.md [hunk 3]
```diff
@@ -9,21 +9,35 @@
 
 [简体中文](README_zh_CN.md)
 
-OpenPAI is an open source platform that provides complete AI model training and resource management capabilities, it is easy to extend and supports on-premise, cloud and hybrid environments in various scale.
+**OpenPAI [v1.0.0](./RELEASE_NOTE.md#2020-v1.0.0) has been released!**
```
Comment (hzy46) [inline README.md:12]: Please change `#2020-v1.0.0` to `#may-2020-version-100`.

Reference: https://github.com/microsoft/pai/blob/pai-0.18.y/RELEASE_NOTE.md#may-2020-version-100

File: README.md [hunk 4]
```diff
@@ -9,21 +9,35 @@
...
+         </tr>
+        <tr>
+         <td colspan="3" align="center"><a href="https://github.com/microsoft/openpaimarketplace"><img src="https://user-images.githubusercontent.com/39592018/82189720-6e7f5d80-9922-11ea-8aa6-ef91bb738f93.png" /></a></td>
+         </tr>
+         <tr>
+          <td align="right">
+             <a href="To be updated"><img src="https://user-images.githubusercontent.com/39592018/82190123-1d239e00-9923-11ea-9278-0c0fc2403bd9.png" align="right" /></a>
+          </td>
+          <td align="center">
+              <a href="https://github.com/microsoft/openpaisdk"><img src="https://user-images.githubusercontent.com/39592018/82190258-5cea8580-9923-11ea-90e3-c69feea89904.png" /></a>
```
Comment (hzy46) [inline README.md:29]: I notice the image is "Python SDK". Actually we don't have a Python SDK now. It is replaced by `openpaisdk`.

File: README.md [hunk 5]
```diff
@@ -9,21 +9,35 @@
 
 [简体中文](README_zh_CN.md)
 
-OpenPAI is an open source platform that provides complete AI model training and resource management capabilities, it is easy to extend and supports on-premise, cloud and hybrid environments in various scale.
+**OpenPAI [v1.0.0](./RELEASE_NOTE.md#2020-v1.0.0) has been released!**
+
+With the release of v1.0, OpenPAI is switching to a more robust, more powerful and lightweight architecture. OpenPAI is also becoming more and more modular so that the platform can be easily customized and expanded to suit new needs. OpenPAI also provides many AI user-friendly features, making it easier for end users and administrators to complete daily AI tasks.
```
Comment (hzy46) [inline README.md:14]: Change v1.0 to `v1.0.0` to remain consistency?

File: README.md [hunk 6]
```diff
@@ -76,7 +90,19 @@ OpenPAI provides end-to-end manuals for both cluster users and administrators.
...
 
-- **Installation**. The installation is based on Kubespray, and here is the [system requirements](https://openpai.readthedocs.io/en/latest/manual/cluster-admin/installation-guide.html#installation-requirements). OpenPAI provides an [installation guide](https://openpai.readthedocs.io/en/latest/manual/cluster-admin/installation-guide.html) to facilitate the installation. 
+- **Installation and upgrade**. The installation is based on Kubespray, and here is the [system requirements](https://openpai.readthedocs.io/en/latest/manual/cluster-admin/installation-guide.html#installation-requirements). OpenPAI provides an [installation guide](https://openpai.readthedocs.io/en/latest/manual/cluster-admin/installation-guide.html) to facilitate the installation. 
+
+If you are considering upgrade from older version to the latest v1.0.0, please refer to the table below for a brief comparison between `v0.14.0` and the upcoming `v1.0.0`. More detail about the upgrade considerations can be found [upgrade guide](https://openpai.readthedocs.io/en/latest/manual/cluster-admin/upgrade-guide.html).
```
Comment (hzy46) [inline README.md:95]: v1.0.0 -> `v1.0.0` (add \`)

File: README.md (hunk 5 above)
Comment (scarlett2018) [reply README.md:14]: v1.0 is more generic, we might have to have v1.0.1 with quick fixes, with this v1.0, we don't need to change it by then...

**Pull request merged**